tunnel-server/src/ssh/base-server.ts

```diff
@@ -94,8 +94,8 @@ export const baseSshServer = (
   const server = new ssh2.Server(
     {
       // debug: x => log.debug(x),
-      // keepaliveInterval: 1000,
-      // keepaliveCountMax: 5,
+      keepaliveInterval: 5000,
```

Comment:
Does it fix the socketserver error thing?
If so, it might worth merging before this PR is ready.